client/web/src/search/results/sidebar/SearchReference.tsx

```diff
@@ -235,20 +245,47 @@ To use this filter, the search query must contain \`type:diff\` or \`type:commit
     },
 ]
 
-for (const searchReference of searchReferenceInfo) {
-    augmentSearchReference(searchReference)
+for (const info of filterInfos) {
+    augmentFilterInfo(info)
 }
 
-const commonFilters = searchReferenceInfo
+const commonFilters = filterInfos
     .filter(info => info.commonRank !== undefined)
     // commonRank will never be undefined here, but TS doesn't seem to know
     .sort((a, b) => (a.commonRank as number) - (b.commonRank as number))
 
+const operatorInfo: OperatorInfo[] = [
+    {
+        operator: 'AND',
+        alias: 'and',
+        placeholder: parsePlaceholder('{a} AND {b}'),
+        description:
+            'Returns results for files containing matches on the left and right side of the `and` (set intersection).',
+        examples: ['conf.Get( and log15.Error(', 'conf.Get( AND log15.Error( AND after'],
+    },
+    {
+        operator: 'OR',
+        alias: 'or',
+        placeholder: parsePlaceholder('{a} OR {b}'),
```

Comment:
In my experience, people are prone to trip themselves up over syntax (what happens when I type `repo:foo a or b` what does the `or` apply to? For this reason I think it would be sensible to add parentheses to the placeholder by default. Your thoughts?

```suggestion
        placeholder: parsePlaceholder('({a} OR {b})'),
```

Reply:
Yep, I like that.